admin/admin-service/src/main/java/io/mosip/admin/dto/BiometricRequestDto.java

```diff
@@ -0,0 +1,19 @@
+package io.mosip.admin.dto;
+
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.util.List;
+
+@Data
+@EqualsAndHashCode
+public class BiometricRequestDto {
+
+    private String id;
+    private String person;

```

Comment:
what is the person property here?

Reply:
you already have a separate property List<String> modalities, what is set in individualBiometrics aka person property?

Reply:
its request @ase-101 we can't change the attribute

Reply:
ok